### PR TITLE
Setup base application

### PR DESCRIPTION
**Pull request by lucasocon**
Add initial base application with Rails 6

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (santiagovidal) [inline config/database.yml]: Usually we ignore this file in `gitignore`.

File: test/application_system_test_case.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```
Comment (santiagovidal) [inline test/application_system_test_case.rb]: I would go ahead and delete everything under `test`. We will use Rspec for testing in the future.

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
-# rs-code-review-metrics
\ No newline at end of file
+<<<<<<< HEAD
+# rs-code-review-metrics
+=======
```
Comment (glm4) [inline README.md]: Watch out 👀

File: README.md [hunk 4]
```diff
@@ -1 +1,28 @@
...
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
+>>>>>>> initial commit
```
Comment (glm4) [inline README.md]: and here

File: Gemfile [hunk 5]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.3'
```
Comment (santib) [inline Gemfile]: it'd be a good idea to leave this version a bit more loose https://andycroll.com/ruby/use-loose-ruby-versioning-in-your-gemfile/

File: Gemfile [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  gem 'capybara', '>= 2.15'
+  gem 'selenium-webdriver'
+  # Easy installation and use of web drivers to run system tests with browsers
+  gem 'webdrivers'
+end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem "pg", "~> 1.1"
```
Comment (santib) [inline Gemfile]: I'd put this above the development and test group

File: config/database.yml (hunk 1 above)
Comment (santib) [inline config/database.yml]: maybe we can remove all sqlite references since we are not going to use it

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore the default SQLite database.
+/db/*.sqlite3
```
Comment (santib) [inline .gitignore]: same here

File: test/application_system_test_case.rb (hunk 2 above)
Comment (santib) [reply test/application_system_test_case.rb]: @lucasocon why was this marked as resolved?

**Pull request merged**